### PR TITLE
makes cloud sample use ADC and simplifies it greatly

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

**Comment by tmatsuo** [comment]
LGTM

File: storage/api/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## Description
+
+This simple command-line application demonstrates how to invoke Google Cloud Storage from PHP.
+
+## Build and Run
+1.  In the [Google Developers Console](https://console.developers.google.com/),
+    create a new project or choose an existing project.
+2.  In the [Google Developers Console](https://console.developers.google.com/),
+    click **APIs & auth**, then click APIs.  Wait for a list of APIs to
```
Comment (JustinBeckwith) [inline storage/api/README.md]: Could we provide a link directly to these?  We do that inside of the nodejs getting started:
https://cloud.google.com/nodejs/getting-started/using-cloud-sql

Check out the "Enable APIs" link.

**Comment by JustinBeckwith** [comment]
LGTM

File: storage/api/README.md (hunk 1 above)
Comment (bshaffer) [inline storage/api/README.md]: Good call. The README steps have been updated.

**Comment by SurferJeffAtGoogle** [comment]
Code changes look good.  Do any documents link to the old code?

**Comment by bshaffer** [comment]
@SurferJeffAtGoogle yes, once it's merged I'll update [this guy](https://cloud.google.com/bigquery/bigquery-api-quickstart)

**Comment by bshaffer** [comment]
oops, I was thinking this was the other PR. For this one I'll need to update [this guy](https://cloud.google.com/storage/docs/json_api/v1/json-api-php-samples)

**Pull request merged**